package.json

```diff
@@ -53,7 +54,7 @@
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
     "react-hot-loader": "^1.3.0",
-    "sinon": "^1.17.1",
+    "sinon": "^2.1.0",
```

Comment:
Does sinon need to be updated?

Reply:
Probably not.  No idea how that happened lol! I also updated all packages for  webpack 3 etc in https://github.com/FormidableLabs/nuka-carousel/pull/261